com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionTransform/PyFunctionTransform_cpp.cgt

```diff
@@ -62,14 +64,27 @@ MY_OPERATOR::MY_OPERATOR()
 
 <% if ($pystyle eq 'python') { %>
 
-    PyObject * depickleInput = 
-      streamsx::topology::Splpy::loadFunction("streamsx.topology.runtime", "depickleInput");
-
+    PyObject * depickleInput = NULL;
     PyObject * funcArg = PyTuple_New(1);
-    PyTuple_SetItem(funcArg, 0, appFunction);
+    
+    <% if ($serializedCallableInstance) { %>
+      depickleInput = streamsx::topology::Splpy::loadFunction("streamsx.topology.runtime", "depickleInputForCallableInstance");
+      PyTuple_SetItem(funcArg, 0, Py_BuildValue("s", <%=$serializedCallableInstance%>));
+      Py_DECREF(appFunction);

```

Comment:
I found trying to understand the use of appFunction hard, it used to be the function that was going to be called (or wrapped by another function), but what is appFunction in the case it's a callable class? What is functionName set to when the function is loaded at line 62?